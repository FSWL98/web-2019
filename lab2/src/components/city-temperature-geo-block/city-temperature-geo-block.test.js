import React from "react";
import CityTemperatureGeoBlock from "./city-temperature-geo-block";
import renderer from 'react-test-renderer';

const cityinfo = {
    icon: "13d",
    name: "Moscow",
    temperature: "10"
};

it("Usual temperature-name geo block", () => {
    const component = renderer.create(<CityTemperatureGeoBlock cityinfo={cityinfo} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
