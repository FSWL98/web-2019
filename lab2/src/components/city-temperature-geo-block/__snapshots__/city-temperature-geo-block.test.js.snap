// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Usual temperature-name geo block 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <p
      className="city-temperature-geo-block__city-name"
    >
      Moscow
    </p>
  </div>
  <div
    className="row"
  >
    <div
      className="col-md-4"
    >
      <img
        alt="icon"
        className="city-temperature-geo-block__icon"
        src="https://openweathermap.org/img/wn/13d@2x.png"
      />
    </div>
    <div
      className="col"
    >
      <span
        className="city-temperature-geo-block__temperature"
      >
        10
        Â°C
      </span>
    </div>
  </div>
</div>
`;
